BarrysPeanutsAkkaFlat/src/main/java/barryspeanuts/actor/CheckOutActor.java

```diff
@@ -0,0 +1,48 @@
+package barryspeanuts.actor;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.AbstractBehavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.actor.typed.javadsl.Receive;
+import java.util.UUID;
+
+public class CheckOutActor extends AbstractBehavior<Object> {
+  private CheckOutActor(ActorContext<Object> context) {
+    super(context);
+  }
+
+  public static Behavior<Object> create() {
+    return Behaviors.setup(CheckOutActor::new);
+  }
+
+  @Override
+  public Receive<Object> createReceive() {
+    return newReceiveBuilder().onMessage(StartCheckout.class, this::handleStartCheckout).build();
+  }
+
+  private Behavior<Object> handleStartCheckout(StartCheckout msg) {
+    getContext()
+        .getLog()
+        .info(
+            "TODO: Getting purchase state for PurchaseId {} from data store and denote that checkout has started.\n",
```

Comment:
Do not need to put newlines in logs. Mentioned before at https://github.com/reselbob/WorkingWithTheActorModel/pull/1#discussion_r1291427243.

Reply:
addressed